### PR TITLE
Add a SQL query for fetching accounts created after a certain date

### PR DESCRIPTION
**Pull request by raskchanky**
tenor-15261355

Closes https://github.com/habitat-sh/habitat/issues/3667
Signed-off-by: Josh Black <raskchanky@gmail.com>

### CODE REVIEW

File: components/builder-sessionsrv/src/migrations/accounts.rs [hunk 1]
```diff
@@ -113,5 +113,26 @@ pub fn migrate(migrator: &mut Migrator) -> SrvResult<()> {
                  ) RETURNS void AS $$
                         DELETE FROM account_origins WHERE account_name=aod_account_name AND origin_id=aod_origin_id;
                  $$ LANGUAGE SQL VOLATILE"#)?;
+
+    // This query is a report that's meant to be run manually. There's no code path (as of
+    // 2017-10-09) that calls it
+    migrator.migrate("accountsrv",
+                     r#"CREATE OR REPLACE FUNCTION account_creation_report (
+                          op_date timestamptz
+                        ) RETURNS TABLE(email text, created_at timestamptz) AS $$
```
Comment (reset) [inline components/builder-sessionsrv/src/migrations/accounts.rs]: We'll need their login, too. Email is now an optional field after we switched to a GitHub App. The user can hide their email from us now

**Comment by reset** [comment]
@raskchanky I think a script to run this on the database and output it into a csv would finish this off. We ideally just want to be able to run this on all shards in a go and then give the csv to a data ingest

**Comment by raskchanky** [comment]
@reset Ah, gotcha.  Will do.

File: components/builder-sessionsrv/src/migrations/accounts.rs (hunk 1 above)
Comment (raskchanky) [reply components/builder-sessionsrv/src/migrations/accounts.rs]: Ok, will add.

**Comment by raskchanky** [comment]
@reset feedback addressed

**Comment by reset** [comment]
tenor-151527693

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/285780072).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/285780072).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**